src/routes/news/components/List.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import React from 'react'
+import { FlatList } from 'react-native'
+import styled from 'styled-components/native'
+import {
+  LocalNewsModel,
+  TunewsModel
+} from '@integreat-app/integreat-api-client'
+
+const Loader = styled.ActivityIndicator`
+  margin-top: 7px;
+`
+const keyExtractor = (item, index) => `${index}`
+
+interface ListPropTypes {
+  items: Array<LocalNewsModel | TunewsModel>,
+  renderItem: ({
+    cityCode: string,
+    language: string,
+    item: LocalNewsModel
+  }) => React$Node,
+  isFetchingMore: boolean,
+  getMoreItems: () => void,
+  renderNoItemsComponent: () => React$Node
+}
+
+const List = ({
+  items,
+  renderItem,
+  isFetchingMore,
+  getMoreItems,
+  renderNoItemsComponent
+}: ListPropTypes) => {
+  return (
+    <FlatList
+      data={items}
+      showsVerticalScrollIndicator={false}
+      contentContainerStyle={{
+        flexGrow: 1,
+        paddingHorizontal: 10
+      }}
+      onEndReached={getMoreItems}
+      ListEmptyComponent={renderNoItemsComponent}
+      ListFooterComponent={isFetchingMore && <Loader size='small' />}
```

Comment:
Doesn't it make sense to use our own LoadingSpinner? What do you think?

Reply:
I actually searched for one but couldn't find. So I will double-check and use your LoadingSpinner

Reply:
I double-checked and there is no common LoadingSpinner component. Should we add a common one?

Reply:
Oh okay, didn't knew that :D However, in `withPayloadProvider` we use react-natives `RefreshControl`. Would that be possible here? If not, I'd favor adding a common component, yes :+1:

Reply:
You mean using the component here or do the same logic as adding refreshControl={}. If the first case, I think this adds nothing to the code whether using ActivityIndicator or RefreshControl. If the second case, it's not possible as this is a different logic. Correct me if I am wrong.

Reply:
Added a common component